changelogs/fragments/338-mysql_user_fix_missing_dynamic_privileges.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "mysql_user: fix missing dynamic privileges after revoke and grant privileges to user (https://github.com/ansible-collections/community.mysql/issues/120)."
```

```suggestion
  - "mysql_user - fix missing dynamic privileges after revoke and grant privileges to user (https://github.com/ansible-collections/community.mysql/issues/120)."
```
for consistency among changelog entries